### ISSUE TITLE
View in-app help on fava public website

### ISSUE THREAD

**Issue by alexjj** [beancount/fava#1044]
Would be nice to be able to review the [in-app help ](https://github.com/beancount/fava/tree/master/fava/help)but via the [fava website.](https://beancount.github.io/fava/index.html)

**Comment by yagebu** [comment]
Are you willing to provide a PR?

**Comment by ueberBrot** [comment]
I would like to try doing this. Is it basically done using all Markdown Files from [/src/fava/help](https://github.com/beancount/fava/tree/master/src/fava/help) ?

From the `.nojekyll` file in it I presume you used [jekyll](https://jekyllrb.com/) for it? I never used jekyll before but from what i found on their website your setup is different?

It would be nice if you could  give me a little bit more information about how you made your gh-pages site.

**Comment by yagebu** [comment]
> Is it basically done using all Markdown Files from /src/fava/help ?

The Markdown files currently expect to be embedded in Fava, so they can link to Fava's reports (which is done in two places). Also, the `url_for` is used to link between the help pages. That would have to be implemented a bit differently for the website.

> From what I found out you are using tox and sphinx to create the website with the MAKEFILE?

Yes, Sphinx is the main tool to build the website. There's just a little script that runs before to generate the sources for the API documentation.

**Comment by ueberBrot** [comment]
> The Markdown files currently expect to be embedded in Fava, so they can link to Fava's reports (which is done in two places). 

Mhh OK. It's not done so easily as I thought. I created a new file in reStructuredText calling it `help.rst` in the `fava/docs` folder.
I made slight adjustments to the `Makefile` to copy the markdown files from `/src/fava/help` in a subdirectory `help` in in the `docs` folder. From there I tried using the [recommonmark](https://www.sphinx-doc.org/en/master/usage/markdown.html) extension for sphinx to include the markdownfiles into the website with the `help.rst` file. That does not work the way I want it to. I get a few errors complaining that the recommonmark extension ins not found but it is included on the site but not in a "pretty" way. To many subheadings and its not rendered the way I would like it to be. 
I'm guessing it has something to do with the problems that recommonmark is not found even though it is installed. I will tinker a bit more when I find more time for it.

> Also, the `url_for` is used to link between the help pages. That would have to be implemented a bit differently for the website.

That is also a thing. The easiest would be just to copy and past it and change it manually but that is bad in many ways. One had to change it in two places if a change would occur. I don't know a way to automate it that it would be change for the use on the gh-pages site. 

If you have any suggestions in how you would do it, I'd like to hear them.
